.github/dependabot.yml

```diff
@@ -1,5 +1,5 @@
 # Please see the documentation for all configuration options:
-# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
```

Comment:
Updates broken link to match module-template

https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/.github/dependabot.yml#L2